aiida_quantumespresso/calculations/pp.py

```diff
@@ -197,6 +209,17 @@ def prepare_for_submission(self, folder):  # pylint: disable=too-many-branches,t
                 parent_calc_folder.computer.uuid,
                 os.path.join(parent_calc_folder.get_remote_path(), self._PSEUDO_SUBFOLDER), self._PSEUDO_SUBFOLDER
             ))
+
+            if copy_files:
+                remote_symlink_list.append((
+                    parent_calc_folder.computer.uuid,
+                    os.path.join(parent_calc_folder.get_remote_path(), f'aiida.in'), 'aiida.ref'
+                ))
```

Comment:
This I need to keep a reference of the `plot_num` used in the `INPUTPP` of the first step because the parser uses this value to decide the units of the `ArrayData` output node. Since it will be retrieved and is small, I could use the `remote_copy_list` to be extra sure but retrieving symlinked files works as well.